src/docgen/render/markdown-doc.ts

```diff
@@ -101,7 +102,7 @@ export class MarkdownDocument {
   /**
    * Render a docs element into the markdown.
    */
-  public docs(docs: DocsSchema) {
+  public docs(docs: DocsSchema, language: Language) {
```

Comment:
This is technically a breaking change...I think it should be optional because the `example` in `DocsSchema` is also optional, and the language is only needed for the example.

```suggestion
  public docs(docs: DocsSchema, language?: Language) {
```

Reply:
Ooh, good catch! 👍